src/main/resources/templates/multi-tool.html

```diff
@@ -177,24 +216,45 @@ <h2 th:text="#{multiTool.header}"></h2>
                 div.classList.add("page-container");
 
                     var img = document.createElement('img');
-                    img.src = await renderer.renderPage(i);
+                    img.classList.add('page-image')
+                    const imageSrc = await renderer.renderPage(i)
+                    img.src = imageSrc;
                     img.pageIdx = i;
                     img.rend = renderer;
                     img.doc = pdfDocument;
                     div.appendChild(img);
 
+                    /**
+                     *  Making pages larger when clicking on them
+                     */
+                    img.addEventListener('mousedown', (x) => {
+                        var bigImg = document.createElement('img');
+                        bigImg.onclick = (e) => {
+                            e.preventDefault();
+                            e.stopPropagation();
+                        };
+                        bigImg.src = imageSrc;
```

Comment:
Correct me if I'm wrong, and yes I'm nit-picking. 
By defining  the callback within the loop of adding images, I think, internally, it creates a new instance of the function each time. If all that is true, it  might be a good idea to move the function somewhere else and access the imageSrc from the .src of the clicked object?